src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Utilities/MediaQuery/BitMediaQueryDemo.razor.scss

```
@@ -0,0 +1,21 @@
+﻿@import '../../../../Styles/abstracts/_bit-css-variables.scss';
```

Stylelint (17.14.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)